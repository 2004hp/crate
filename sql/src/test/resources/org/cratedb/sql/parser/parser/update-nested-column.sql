update t set a['b']['c']['d'] = 3