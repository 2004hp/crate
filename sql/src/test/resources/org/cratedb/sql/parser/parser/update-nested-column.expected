org.cratedb.sql.parser.parser.UpdateNode@5144f423
statementType: UPDATE
resultSet: 	
	org.cratedb.sql.parser.parser.SelectNode@5f84f84
	isDistinct: false
	resultColumns: 		
		org.cratedb.sql.parser.parser.ResultColumnList@19e7cb19

		[0]:		
		org.cratedb.sql.parser.parser.ResultColumn@41883161
		exposedName: a.b.c.d
		name: a.b.c.d
		tableName: null
		isDefaultColumn: false
		type: null
		expression: 			
			org.cratedb.sql.parser.parser.NumericConstantNode@6c1b2a5c
			value: 3
			type: INTEGER NOT NULL
		reference: 			
			org.cratedb.sql.parser.parser.NestedColumnReference@6b7a3576
			columnName: a.b.c.d
			tableName: null
			type: null
			path:
				[0]:				
				org.cratedb.sql.parser.parser.CharConstantNode@244c0d1a
				value: b
				type: CHAR(1) NOT NULL
				[1]:				
				org.cratedb.sql.parser.parser.CharConstantNode@7cf03893
				value: c
				type: CHAR(1) NOT NULL
				[2]:				
				org.cratedb.sql.parser.parser.CharConstantNode@7f127af9
				value: d
				type: CHAR(1) NOT NULL
	fromList: 		
		org.cratedb.sql.parser.parser.FromList@1647b29c

		[0]:		
		org.cratedb.sql.parser.parser.FromBaseTable@3d1f2b31
		tableName: t
		updateOrDelete: UPDATE
		null
		correlation Name: null
		null
targetTableName: 	
	org.cratedb.sql.parser.parser.TableName@74
	t